apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
```

Comment:
Do you know who owns this placeholder video? I think we should try to source our own. Brand Studio may have something that we can use.

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)